code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -25,17 +25,17 @@
 	var/tiers_to_pick_from
 	switch(tier)
 		if(XENO_TIER_ZERO, XENO_TIER_FOUR)
-			to_chat(src, "<span class='warning'>Your tier does not allow you to regress.</span>")
-			return
+			if(isxenoshrike(src))
+				tiers_to_pick_from = GLOB.xeno_types_tier_one
+			else
+				to_chat(src, "<span class='warning'>Your tier does not allow you to regress.</span>")
+				return
```

Comment:
early return?
plus don't return at the end of a else in a switch, it's useless

Reply:
Look at the surrounding code, I just moved the special shrike check from below, the else is already inside of a switch case and since DMcode does not support break for switches I could not early break instead.